beefy-gadget/src/gossip.rs

```diff
@@ -72,6 +72,9 @@ where
 		}
 	}
 
+	/// Note a live voting round
+	///
+	/// This should be called, in order to keep tabs on `round`.
```

Comment:
What do you mean by "keep tabs"?

Reply:
`to keep tabs on sth` means to observe / control / keep sth under observation.

But this comment is going to change with the upcoming gossip fix anyways.